src/services/CertificateGenerationService.ts

```diff
@@ -62,10 +46,16 @@ class CertificateGenerationService {
             hgv_prs: config.documentNames.hgv_prs,
             trl_pass: config.documentNames.vtg5a,
             trl_fail: config.documentNames.vtg30,
-            trl_prs: config.documentNames.trl_prs
+            trl_prs: config.documentNames.trl_prs,
+            rwt: config.documentNames.rwt
         };
 
-        const vehicleTestRes: string = testResult.vehicleType + "_" + testType.testResult;
+        let vehicleTestRes: string;
+        if (CertificateGenerationService.isRoadworthinessTestType(testType.testTypeId)) { // CVSB-7677 is roadworthisness test
+            vehicleTestRes = "rwt";
+        } else  {
+        vehicleTestRes = testResult.vehicleType + "_" + testType.testResult;
```

Comment:
linting should have complained about tabulation

Reply:
verified with npm run tslint. Its not throwing an errors.